docs/src/main/asciidoc/resource-handling.adoc

```diff
@@ -1,5 +1,8 @@
 == Resource Handling
 
+This article describes using Spring Cloud Azure Starters `spring-cloud-azure-starter-storage-blob` and `spring-cloud-azure-starter-storage-file-share` to auto-configure Azure Storage Blob and Azure Storage File Share in your Spring projects. It implements Spring Resource abstraction for Azure Storage service which allows you to interact with Azure storage Blob and File Share using Spring programming model.
```

Comment:
Why do we use this article here?

Reply:
I don't think that's the only functionality these two starters provide, for they also support the auto-configuration of storage clients. Could you take a look at this https://docs.awspring.io/spring-cloud-aws/docs/2.3.0-RC1/reference/html/index.html#resource-handling?

Reply:
> Why do we use this article here?

This is from the [readme](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/spring/azure-spring-boot-starter-storage/README.md), why is "this article" not "The project" or others, because in the part "Resource handling",  from this perspective, "this article" maybe more suitable.

Reply:
> I don't think that's the only functionality these two starters provide, for they also support the auto-configuration of storage clients. Could you take a look at this [docs.awspring.io/spring-cloud-aws/docs/2.3.0-RC1/reference/html/index.html#resource-handling](https://docs.awspring.io/spring-cloud-aws/docs/2.3.0-RC1/reference/html/index.html#resource-handling)?

Sorry, I didn't get your point.

Reply:
Can we find a better word than this article?

Reply:
> > I don't think that's the only functionality these two starters provide, for they also support the auto-configuration of storage clients. Could you take a look at this [docs.awspring.io/spring-cloud-aws/docs/2.3.0-RC1/reference/html/index.html#resource-handling](https://docs.awspring.io/spring-cloud-aws/docs/2.3.0-RC1/reference/html/index.html#resource-handling)?
> 
> Sorry, I didn't get your point.

Are we on the same page right now?